defaults/main.yml

```diff
@@ -1,12 +1,23 @@
 ---
+openio_kibana_namespace: "OPENIO"
+openio_kibana_serviceid: "0"
+
+openio_kibana_gridinit_dir: "/etc/gridinit.d"
+openio_kibana_gridinit_file_prefix: "{{ openio_kibana_namespace }}-"
+
+openio_kibana_sysconfig_dir: "/etc/oio/sds/{{ openio_kibana_namespace }}/{{ openio_kibana_servicename }}"
+openio_kibana_log_dir: "/var/log/oio/sds/{{ openio_kibana_namespace }}/{{ openio_kibana_servicename }}"
+openio_kibana_pid_dir: "/run/oio/sds/{{ openio_kibana_namespace }}"
+
+openio_kibana_volume:
+  "/var/lib/oio/sds/{{ openio_kibana_namespace }}/kibana-{{ openio_kibana_serviceid }}"
+
 openio_kibana_provision_only: false
 openio_kibana_service_enabled: true
 openio_kibana_bind_port: 5601
-openio_kibana_confdir: "/etc/kibana"
 openio_kibana_bind_interface: "{{ openio_bind_mgmt_interface | d(ansible_default_ipv4.alias) }}"
-openio_kibana_bind_address: "{{ openio_bind_mgmt_address \
-      | default(hostvars[inventory_hostname]['ansible_' + openio_kibana_bind_interface]['ipv4']['address']) }}"
-openio_kibana_ext_bind_address: ""
+openio_kibana_bind_address: "{{ openio_bind_mgmt_address
+  | d(hostvars[inventory_hostname]['ansible_' + openio_kibana_bind_interface].ipv4.address) }}"
```

Comment:
| default(hostvars[inventory_hostname]['ansible_' + openio_kibana_bind_interface]['ipv4']['address']

Don't mix format
Choose the dot style or array style, not both

Reply:
fixed